rholang/src/test/scala/coop/rchain/rholang/interpreter/ReduceSpec.scala

```diff
@@ -80,8 +81,8 @@ class ReduceSpec extends FlatSpec with Matchers with AppendedClues with Persiste
     )
 
   "evalExpr" should "handle simple addition" in {
-    implicit val errorLog = new ErrorLog[Task]()
-    val result = withTestSpace(errorLog) {
+    implicit val error = ErrorHandling.emptyError[Task].runSyncUnsafe()
```

Comment:
Adding more `runSyncUnsafe`'s decreases readability. Is it a lot of effort to get rid of them?

Does the `val error` need to be implicit if it's passed in explicitly on the line below?

Reply:
No. It does not need to be implicit. But the next few commits eliminate `error` in the tests, so the correction is unnecessary.

Reply:
What about the `runSyncUsafe`s?

Reply:
I see what you mean. I'll remove them.